src/main/java/util/RandomDirectionGenerator.java

```diff
@@ -0,0 +1,17 @@
+package util;
+
+import domain.ladder.common.Direction;
+
+import java.util.Random;
+
+public class RandomDirectionGenerator implements DirectionGenerator {
+    @Override
+    public Direction generate(){
+        Random random = new Random();
+        boolean flag = random.nextBoolean();
+        if(flag){
```

Comment:
```suggestion
    public Direction generate() {
        Random random = new Random();
        boolean flag = random.nextBoolean();
        if (flag) {
```
인텔리제이에서 제공하는 코드포맷팅을 적절히 사용하시면 좋을 것 같네요

Reply:
자동 적용이 된 줄 알았는데 놓친 부분이 있었습니다! 신경쓰도록 하겠습니다ㅎㅎ